apps/base-docs/tutorials/docs/5_shopify-storefront-commerce.md

```diff
@@ -0,0 +1,99 @@
+---
+title: 'Deploy a Shopify Storefront with Coinbase Commerce'
+slug: /shopify-storefront-commerce
+description: Learn how to launch a Shopify storefront that uses Coinbase Commerce as a crypto payment gateway.
+author: hughescoin
+keywords:
+  [Shopify, Coinbase Commerce, crypto payments, ecommerce, Hydrogen, Oxygen, USDC, ocs, onchain]
+tags: ['ecommerce', 'crypto', 'shopify', 'onchain summer', ocs]
+difficulty: easy
+hide_table_of_contents: false
+displayed_sidebar: null
+---
+
+Deploy a Shopify Storefront with Coinbase Commerce Plugin
+[Shopify](https://www.shopify.com/) provides a robust platform for creating online stores. This tutorial will guide you through setting up a Shopify storefront that integrates with Coinbase Commerce, allowing you to accept cryptocurrency payments.
+
+## Objectives
+
+By the end of this tutorial, you will have a custom storefront with integrated crypto payments, using your existing Shopify products.
+
+You will learn how to:
+
+- Create a Shopify storefront
+- Link your Coinbase Commerce account to Shopify
+- Link Shopify credentials to your storefront
+- Deploy a test site using Hydrogen/Oxygen
+
+## Prerequisites
+
+- Shopify Basic or better
+- Github account
+- Business Coinbase Commerce account | [Sign up](https://beta.commerce.coinbase.com/sign-up)
+
+## Getting Started
+
+### Link Coinbase Commerce to your Shopify store
+
+![shopify-install-commerce.gif](../../assets/images/shopify-storefront-commerce/shopify-install-commerce.gif)
+
+1. Navigate to your admin page: (`https://admin.shopify.com/store/<YOUR-STORE-NAME>`)
+2. Click on Settings (`https://admin.shopify.com/store/<YOUR-STORE-NAME>settings/general`)
+3. Click on the “Payments” tab
+4. Click “Add a payment method”
+5. Click “Search by provider”
+6. Type in “Coinbase” in the search field
+7. Select “Coinbase Commerce”
+8. Click "Install"
+9. Log into your Coinbase account
```

Comment:
Trying to sign up for a business account is just taking me through a login process and dumping me onto coinbase.com/home

Either we've got a bug, or it should be clearer that you **can not** use the email you have a retail account with for a business account.  Can you please check with them and update as appropriate?